osu.Game/Screens/SelectV2/BeatmapCarousel.cs

```diff
@@ -397,7 +399,10 @@ protected override Drawable GetDrawableForDisplay(CarouselItem item)
         {
             switch (item.Model)
             {
-                case GroupDefinition:
+                case GroupDefinition group:
+                    if (group.Data is int)
```

Comment:
This is super haphazard. Why not use [`StarDifficulty`](https://github.com/ppy/osu/blob/d37c1bb6d04269f32d9739dcc74fb781bb937358/osu.Game/Beatmaps/StarDifficulty.cs#L9)?